content/blog/2023/when-to-choose-a-native-mobile-app.md

```diff
@@ -122,42 +139,44 @@ Native mobile apps require a unique codebase for each platform. For example, if
 you want to make the same native mobile app available on both Android and iOS
 devices, you need to build one codebase for Android and a separate codebase for
 iOS. Each codebase requires different proprietary technologies, and likely
-requires separate teams of developers to build and maintain for each platform.
-It is frequently more expensive to reach the same range of people with a native
-mobile app compared to an RWA.
+requires separate teams of developers to build and maintain each platform. It is
+frequently more expensive to reach the same range of people with a native mobile
+app compared to a responsive web app.
 
 ### Lengthy Development
 *Don't choose a native mobile app when it's a priority to make your app
 available to users quickly.*
 
 Because native mobile apps require a separate codebase for each platform you
 want to support, development generally takes longer than it does to develop a
-single codebase for an RWA. Additionally, each app store has its own set of
-requirements in order to publish an app, and these can be complex. Wait times
-for app store approval to publish or update the app can be lengthy as well.
+single codebase for a responsive web app. Additionally, each app store has its
+own set of requirements in order to publish an app, and these can be complex.
+Wait times for app store approval to publish or update the app can be lengthy as
+well.
 
 ### Not Discoverable Via Search Engines
 *Don't choose a native mobile app when the people you want to reach will look
 for your content with a search engine.*
 
-Unlike for RWAs, search engines don't index native mobile apps. If your research
-indicates that people frequently use search engines to search for the type of
-content you offer, an RWA may be a better choice. Alternatively, you could build
-a marketing website for search engines to find, which simply introduces your app
-and links people to the appropriate app store for download. (A separate
-marketing website will further increase the costs and development costs.)
+Unlike for responsive web apps, search engines don't index native mobile apps.
+If your research indicates that people frequently use search engines to search
+for the type of content you offer, an RWA may be a better choice. Alternatively,
+you could build a marketing website for search engines to find, which simply
+introduces your app and links people to the appropriate app store for download.
+(A separate marketing website will further increase the design and development
+costs.)
```

Comment:
My leaning is to not go into that here because I think the people who will read this article are unlikely to be building something on the scale of Facebook or NYTimes, but perhaps I shouldn't assume.